### DISCUSSION TITLE
Resolution and loading problems

### DISCUSSION THREAD

**Discussion by 73N1X** [brian-mckeown/flightdash#69] [Beta Testers]
Some minor problems adjusting to 2K resolution, maybe not responsive? Inner content doesn't adjust leaving blank spaces. 

![Captura de pantalla 2024-02-16 030854](https://github.com/brian-mckeown/flightdash/assets/115502401/228e3cfd-1bb0-40c4-bcf0-ac2831080274)

Navigating between pages using the menu is causing some loading problems, the page freezes.

**Comment by brian-mckeown** [comment]
Thanks for the feedback. Yes, we are working to improve the responsive design of the vat track map within the EFB app. It certainly does appear less appealing in that screenshot with a wider monitor. I'll go ahead and create an issue for tracking.

As far as the loading/freezing issues.. There may be an initial load time for the map, but afterwards there should not be an issue, especially just navigating through the different pages. Are you able to let me know what specific clicks are causing the issue? The vatsim data refreshes in the background every 20seconds. Let me know if you see a pattern of performance issue related to that 20sec interval. 

Thanks!

**Reply by brian-mckeown** [reply]
https://github.com/brian-mckeown/flightdash/issues/70

**Reply by 73N1X** [reply]
Performance issue must be related to vatsim data retrieval, you can navigate freely between pages in the side-menu but every 20 seconds the page freezes and take ~5 seconds to recover and give you the chance to navigate again.

**Reply by brian-mckeown** [reply]
Thanks for confirming. This is a known issue discovered after our last update for adding center controller airspace to the map. A fix will be in place for this soon.

**Comment by brian-mckeown** [comment]
Just letting you an update was pushed this morning (v1.6.4) that morning that should significantly improve the loading/refresh time. To try it out, be sure to clear your browser cache and ensure v1.6.4 is showing up at the bottom-right corner of the app. Thanks again for the feedback!
